client/src/components/Tombstones/Tombstones.tsx

```diff
@@ -24,17 +24,19 @@ interface LabeledTombstoneProps {
 }
 
 const LabeledTombstone = ({ labels_and_items }: LabeledTombstoneProps) => (
-  <dl className="row tombstone-data-list">
+  <div>
     {_.map(labels_and_items, ([label, item], ix) => (
       <Fragment key={ix}>
-        <dt className="col-12 col-lg-2">{label}</dt>
-        <dd className="col-12 col-lg-10">{item}</dd>
+        <dl className="row tombstone-data-list">
+          <dt className="col-12 col-lg-2">{label}</dt>
+          <dd className="col-12 col-lg-10">{item}</dd>
+        </dl>
         {ix !== _.size(labels_and_items) - 1 && (
           <hr style={{ width: "100%" }} />
         )}
       </Fragment>
     ))}
-  </dl>
+  </div>
```

Comment:
No screen reader should be announcing an `hr` tag, as far as I know. I expect this was a false-positive. Does leaving it as-was and marking the `hr` as `aria-hidden` silence the alert?

I think the screen reader experience for individual `dl`s per `dt`/`dd` pair will be much different/worse than a single `dl`. Double check that please. If so, and if slapping `aria-hidden` on the `hr` isn't enough, you might need to recreate the horizontal line as pure CSS (it'll be a bother, take a couple media queries and potentially create a brittle sync between the column classes on these elements and the component's style sheet).

Reply:
You're right about the screen reader not announcing any `hr` tag. When I had tested the difference between the two implementations, the only difference was that having the paired up elements allowed for selecting and reading the full row rather than just either the `dt` or `dd` tag.

Reply:
Oh sorry, hadn't seen this... two implementations being the one that was already in the code and then my solution. My solution allowed for selecting the full rows.

Reply:
Hm, unfortunately every screen reader probably handles that differently. As much as possible, we need to respect the semantics of a tag as described in the spec, since it'd be impractical to test with (and satisfy all) different screen reader implementations. Doing something iffy because it works better for Mac voice over probably makes things worse for other, unknown, screen readers :shrug:.

This is iffy because the `dl` is explicitly a (description) list. X number of sibling, single element, lists is semantically different from one list with X elements.

[Here's something helpful](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#wrapping_name-value_groups_in_div_elements): per the spec, the `dt`/`dd` pairs can be wrapped in `div`s. That'll make replacing the `hr` with a bottom border style much more trivial.

Side note, that mozilla doc page [also mentions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#accessibility_concerns) that every screen reader is going to express them in their own special ways, haha.